config.schema.yaml

```diff
@@ -0,0 +1,16 @@
+---
+  host:
+    description: "Hostname or IP address of the Redis instance"
+    type: string
+    secret: false
+    required: true
+  port:
+    description: "Port of the Redis instance"
+    type: integer
+    default: 6379
+    required: true
+  database:
```

Comment:
It would also be good to add support for password / authentication, but not a blocker for now.